pipdeptree.py

```diff
@@ -418,6 +466,12 @@ def main():
                             '"raw" output that may be used by external tools. '
                             'This option overrides all other options.'
                         ))
+    parser.add_argument('--graph-output', dest='output_format',
```

Comment:
Good point about having to add graphviz as mandatory dependency. Let's not use choices.